src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -15,7 +15,6 @@ var DOMProperty = require('DOMProperty');
 
 var MUST_USE_PROPERTY = DOMProperty.injection.MUST_USE_PROPERTY;
 var HAS_BOOLEAN_VALUE = DOMProperty.injection.HAS_BOOLEAN_VALUE;
-var HAS_SIDE_EFFECTS = DOMProperty.injection.HAS_SIDE_EFFECTS;
```

Comment:
Want to prep the followup PR to remove the `HAS_SIDE_EFFECTS` logic? Or just do it here as a 2nd commit (I think the reasoning for a followup in the other PR was that it was a non-core contributor).